RecoTracker/MkFit/plugins/DumpMkFitGeometry.cc

```diff
@@ -0,0 +1,57 @@
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
```

Comment:
this can probably be in the /test as well (the review is less demanding)

Reply:
You mean DumpMkFitGeometry.cc? How does it get put into the plugin library then? Or we need a separate lib? @makortel what do you think?

Reply:
If the module is not intended to be run in production, `/test` would be better place (reduces code bloat for production). An additional plugin library needs to be declared in  `test/BuildFile.xml`, something along
```xml
<library file="DumpMkFitGeometry.cc" name="RecoTrackerMkFitTest">
  <flags EDM_PLUGIN="1"/>
  <use name="FWCore/Framework"/>
  <use name="RecoTracker/MkFit"/>
  <use name="RecoTracker/MkFitCore"/>
  <use name="RecoTracker/Record"/>
</library>
```

Reply:
ok, this works ... moved.